### DISCUSSION TITLE
about external gcs

### DISCUSSION THREAD

**Discussion by jakauser** [RosettaDrone/rosettadrone#169] [Q&A]
Why is rosetta drone interlocking external gcs? 
Which part of the essential features of the dji drone as a gcs is not implemented?

**Comment by kripper** [comment]
Because RosettaDrone is not focused on developing a GCS, but only the MAVLink compatibility layer required to support the best existing GCS.

Maybe we have some GCS like features, but very limited and unmaintained. For example, the internal waypoints editor doesn't even support the MAVLINK data dictionary, but only a subset of the DJI waypoints specification which is only supported on some DJI models.

If we wanted to have a full fledged GCS, we would better do some integration with QGC for Android.
